azure-pipelines.yml

```diff
@@ -76,6 +76,10 @@ jobs:
           yarn bundle --no-cache $(sinceArg)
         displayName: bundle
 
+      - script: |
+          yarn lage build-storybook --no-cache $(sinceArg) --scope @fluentui/react-components
```

Comment:
for now execute only on react-components if affected -> follow up https://github.com/microsoft/fluentui/pull/19366